include/elf.h

```diff
@@ -182,6 +182,8 @@ typedef struct mips_elf_abiflags_v0 {
 
 #define EM_NANOMIPS     249     /* Wave Computing nanoMIPS */
 
+#define EM_LOONGARCH    258     /* Loongarch */
```

Comment:
Like [binutils](https://sourceware.org/git/?p=binutils-gdb.git;a=blobdiff;f=include/elf/common.h;h=24d0a09b7c8f74558aaabbce295e19792f264b6d;hp=234aef084b2f2e84caa86de896029fd0e97d9b56;hb=01a8c731aacbdbed0eb5682d13cc074dc7e25fb3;hpb=8dc196f2fa8e2ea5a0fd6aab58475aa88dd3c45e), the comment should be LoongArch.